### ISSUE TITLE
05_Composing_Plots.ipynb raises ValueError: Index is not contained within new index

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/holoviz#256]
HoloViz:  0.11.3
Dask:  2.17.2
HvPlot: 0.5.2
HoloViews: 1.13.2
OS: Win 10

# My Pain

Preparing for a course on HoloViz that I will be giving. When I run the 05_Composing_Plots.ipynb notebook I get the below error

![image](https://user-images.githubusercontent.com/42288570/83397704-5c132280-a3fe-11ea-8a53-ef5398783f30.png)

```bash
...

~\AppData\Local\Continuum\anaconda3\envs\holoviz-tutorial\lib\site-packages\dask\core.py in _execute_task(arg, cache, dsk)
    119         # temporaries by their reference count and can execute certain
    120         # operations in-place.
--> 121         return func(*(_execute_task(a, cache) for a in args))
    122     elif not ishashable(arg):
    123         return arg

~\AppData\Local\Continuum\anaconda3\envs\holoviz-tutorial\lib\site-packages\dask\dataframe\tseries\resample.py in _resample_series(series, start, end, reindex_closed, rule, resample_kwargs, how, fill_value, how_args, how_kwargs)
     48     if not out.index.isin(new_index).all():
     49         raise ValueError(
---> 50             "Index is not contained within new index. This can often be "
     51             "resolved by using larger partitions, or unambiguous "
     52             "frequencies: 'Q', 'A'..."

ValueError: Index is not contained within new index. This can often be resolved by using larger partitions, or unambiguous frequencies: 'Q', 'A'...
```

#### Additional Context

I'm not normally a user of Dask so I not capable of understanding this error message and if it is really a Dask, HvPlot or Data issue.

**Comment by MarcSkovMadsen** [comment]
#### Workaround

After some experimentation i discovered that I could change `1M` to `1W`.

![image](https://user-images.githubusercontent.com/42288570/83399023-c62cc700-a400-11ea-90ca-511ce1e57a85.png)

**Comment by MarcSkovMadsen** [comment]
![image](https://user-images.githubusercontent.com/42288570/83399081-e066a500-a400-11ea-9c38-bba8d7e68bbb.png)

**Comment by philippjfr** [comment]
Btw, thanks for going through all this. I'm pretty dismayed at the state here. I was **never** a fan of the metapackage but having just discovered that it doesn't even pin versions properly it's worse than useless so my opinion is that we should get rid of this asap and/or at least provide one which pins very explicitly to versions known to be working. My plan was to release hvPlot today and then update the tutorial notebooks.

**Comment by jbednar** [comment]
Right; somehow the metapackage setup was declaring appropriate pinning, but it wasn't being used. I've fixed that in #261, and also applied your selected fix to change from monthly to weekly indexing. I must admit that I have no idea why that should matter, but it does fix it! Thanks.
